src/gui/qgsmaplayerstylecategoriesmodel.cpp

```diff
@@ -274,6 +274,18 @@ QVariant QgsMapLayerStyleCategoriesModel::data( const QModelIndex &index, int ro
       }
       break;
 
+    case QgsMapLayer::StyleCategory::Elevation:
+      switch ( role )
+      {
+        case Qt::DisplayRole:
+          return tr( "Elevation Properties" );
+        case Qt::ToolTipRole:
+          return tr( "Elevation properties" );
+        case Qt::DecorationRole:
+          return QIcon(); // TODO
```

Comment:
@nyalldawson , I'll try to come up with something in the coming week or so. I don't think we should hold off the PR for icon though.